src/gnatss/harmonic_mean.py

```diff
@@ -71,6 +73,8 @@ def sv_harmonic_mean(svdf: pd.DataFrame, start_depth: float, end_depth: float) -
     abs_sv = abs(svdf)
     # Get the index for the start of depth closest to specified start depth
     if len(abs_sv) == 0:
-        raise ValueError("Dataframe is empty! Please check your data inputs.")
+        raise ValueError(
+            "DataFrame with empty 'abs_sv' column! Please check your data inputs."
```

Comment:
`abs_sv` is the same exact dataframe but values are the absolute value, not an empty 'abs_sv' column. If you see line 73 above, it wraps `svdf` with `abs` function. You've caught this with your new change in `svdf.empty` so probably don't need this check anymore of `if len(abs_sv)`... does that makes sense?

Reply:
Thanks for noting this redundancy issue, I am sorry for didn't notice this. I will also delete the redundancy test case in the test file too.